CONTRIBUTING.md

```diff
@@ -379,12 +379,88 @@ The following limitations are recommended:
 
 ### Observability
 
-Out of the box, your users should be able to observe the state of your component.
-See [observability.md](docs/observability.md) for more details.
+Out of the box, your users should be able to observe the state of your
+component. See [observability.md](docs/observability.md) for more details.
 
 When using the regular helpers, you should have some metrics added around key
-events automatically. For instance, exporters should have `otelcol_exporter_sent_spans`
-tracked without your exporter doing anything.
+events automatically. For instance, exporters should have
+`otelcol_exporter_sent_spans` tracked without your exporter doing anything.
+
+Custom metrics can be defined as part of the `metadata.yaml` for your component.
```

Comment:
it would be good to point to some concrete examples, e.g. ./receiver/receiverhelper/metadata.yaml